src/gnome-shell-extension-prefs.c

```diff
@@ -28,6 +28,7 @@ main (int argc, char **argv)
     {
       g_message("Failed to defined ARGV: %s", error->message);
       g_error_free (error);
+      g_object_unref (context);
```

Comment:
I'm not sure if we're using it here, but if we have `g_autoptr` it seems like this is a good candidate for its use.